### PR TITLE
Episode 1 - A new layout

### PR DESCRIPTION
**Pull request by mr-mr**
The mother of all PR's for me.
This started as a simple 'move the form elements out the left hand column' and has snowballed into an enormous series of commits.

What it looks like now
image

What's proposed
image

There's little niggle and tweaks to be made (empty states needs some love. But it's so big now we can solve those after this is merged in.

cc @Reettaphant @mbarton @jennysivapalan @akash1810 @ShaunYearStrong

### CODE REVIEW

**Comment by Reettaphant** [comment]
The build is still failing due to some lint errors I think.

**Comment by ShaunYearStrong** [comment]
Yep, looks like you forgot to delete the old module imports. There are a couple of others too. Do you have access to CircleCi to take a look at them?

File: public/video-ui/src/components/Video/Display.js [hunk 1]
```diff
@@ -3,51 +3,78 @@ import VideoEdit from '../VideoEdit/VideoEdit';
...
 
   cannotEditStatus = () => {
     return this.props.video.expiryDate <= Date.now();
   };
 
+  renderEditButton = (editable, property) => {
+
+    if (editable) {
+      return (
+        <i className="icon icon__done" onClick={this.manageEditingState.bind(this, false, property)}>done</i>
```
Comment (ShaunYearStrong) [inline public/video-ui/src/components/Video/Display.js]: "Done editing" perhaps? It'll be a little more explicit for screen reader users.

File: public/video-ui/src/components/Video/Display.js [hunk 2]
```diff
@@ -60,27 +87,50 @@ class VideoDisplay extends React.Component {
...
           <div className="video__main">
             <div className="video__main__header">
               <div className="video__detailbox">
                 <span className="video__detailbox__header">Preview</span>
                 <VideoPreview video={this.props.video || {}} />
               </div>
-              <div className="video__detailbox usages">
+              <div className="video__detailbox">
+                <div className="video__detailbox__header__container">
+                  <span className="video__detailbox__header">Video Meta Data</span>
```
Comment (ShaunYearStrong) [inline public/video-ui/src/components/Video/Display.js]: Is a `<span>` appropriate? Marking it up as a header would make more semantic sense.

File: public/video-ui/src/components/Video/Display.js [hunk 3]
```diff
@@ -60,27 +87,50 @@ class VideoDisplay extends React.Component {
...
+                  video={this.props.video || {}}
+                  saveVideo={this.saveVideo}
+                  saveAndUpdateVideo={this.saveAndUpdateVideo}
+                  disableStatusEditing={this.cannotEditStatus()}
+                  editable={this.state.youtubeEditable}
+                />
+              </div>
+              <div className="video__detailbox">
+                <div className="video__detailbox__header__container">
+                  <span className="video__detailbox__header">Poster Image</span>
```
Comment (ShaunYearStrong) [inline public/video-ui/src/components/Video/Display.js]: As above

File: public/video-ui/src/components/VideoAssets/VideoAssets.js [hunk 4]
```diff
@@ -115,7 +115,7 @@ class VideoAssets extends React.Component {
         <div className="video-assets">
           <div className="section-header">
             <span className="video__detailbox__header">Assets</span>
```
Comment (ShaunYearStrong) [inline public/video-ui/src/components/VideoAssets/VideoAssets.js]: Make this a header tag

File: public/video-ui/src/components/VideoAssets/VideoAssets.js [hunk 5]
```diff
@@ -115,7 +115,7 @@ class VideoAssets extends React.Component {
...
-            {!this.state.showAssetForm ? <button className="btn section-header__btn" type="button" onClick={this.showAssetForm}>Add new asset</button> : false}
+            {!this.state.showAssetForm ? <button type="button" onClick={this.showAssetForm}><i className="icon icon__edit">add</i></button> : false}
```
Comment (ShaunYearStrong) [inline public/video-ui/src/components/VideoAssets/VideoAssets.js]: "Add asset" would be more explicit

File: public/video-ui/src/constants/videoEditValidation.js [hunk 6]
```diff
@@ -1,5 +1,5 @@
 const validate = (values) => {
```
Comment (ShaunYearStrong) [inline public/video-ui/src/constants/videoEditValidation.js]: I believe in this file there shoudl also be a character limit on `title`. @akash1810 - What is the limit?

File: public/video-ui/styles/components/_detail.scss [hunk 7]
```diff
@@ -9,6 +9,9 @@
     &__item {
       padding: 5px 10px;
       border-top: 1px solid $color600Grey;
+      width: 100%;
+      overflow: hidden;
```
Comment (ShaunYearStrong) [inline public/video-ui/styles/components/_detail.scss]: I've a feeling this is going to annoy people. We can leave it in for now though.

File: public/video-ui/styles/components/_details-list.scss [hunk 8]
```diff
@@ -1,7 +1,8 @@
 .details-list__title {
   font-weight: bold;
   margin: 0 0 5px;
-  padding-right: 5px;
+  padding: 8px 0 0;
```
Comment (ShaunYearStrong) [inline public/video-ui/styles/components/_details-list.scss]: 8px is a bit magical but I'll allow it.

File: public/video-ui/styles/layout/_video.scss [hunk 9]
```diff
@@ -67,17 +59,21 @@
   width: 300px;
 }
 
+.video__detailbox__header__container {
+  display: flex;
+  justify-content: space-between;
+}
+
 .video__detailbox {
+  width: 23vw;
```
Comment (ShaunYearStrong) [inline public/video-ui/styles/layout/_video.scss]: veeery magical. As this is for layout, there's probably a more mathematical approach to this.

Comment (mr-mr) [inline public/video-ui/styles/layout/_video.scss]: Agreed, I should've looked again before commiting.

**Comment by jennysivapalan** [comment]
Late to the party but just wanted to add that this looks great! 🎉 
On the `/videos/:id/audit` should we have a link back to `video/:id` page? You can get there by browser back but just wondering if we should have something visual on the page to link back?

**Comment by ShaunYearStrong** [comment]
@jennysivapalan - Good UX skills! I agree on the link, definitely.

**Pull request merged**